src/index.html

```diff
@@ -1,13 +1,560 @@
+              </p>
+              <br>
+              <p class="about__info  about__info--time">
+              Closed on
+              </p>
+              <p class="about__info">
+              Thanksgiving Day, Dec 25, Jan 1, and the first Monday in May.
+              </p>
+
+              <p class="about__info about__info--main">
+              The Met Cloisters
+              </p>
+              <br>
+              <p class="about__info  about__info--time ">
+              10 am – 4.30 pm
+              </p>
+              <p class="about__info">
+              Thu - Tue
+              </p>
+              <br>
+              <p class="about__info  about__info--time">
+              Closed
+              </p>
+              <p class="about__info">
+              Wed
+              </p>
+              <br>
+              <p class="about__info about__info--time">
+              Closed on
+              </p>
+              <br>
+              <p class="about__info">
+              Thanksgiving Day, Dec 25, Jan 1
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section gallery" id="gallery">
+        <div class="container">
+          <h2 class="page__section-title">
+            Now On View
+          </h2>
+          <p class="page__section-info">
+            Explore exhibition gallery guides, videos, articles, and more.
+          </p>
+          <div class="gallery__place art">
+            <div class="art__photos">
+              <div class="grid">
+                <div
+                  class="grid__item
+                         grid__item--desktop-1-9
+                         grid__item--tablet-1-5"
+                >
+                  <div class="art__photo-container">
+                    <img
+                      src="images/art/1.jpg"
+                      alt="Art photo 1"
+                      class="art__photo"
+                    >
+                  </div>
+                </div>
+                <div
+                  class="grid__item
+                         grid__item--desktop-9-13
+                         grid__item--tablet-5-7"
+                >
+                  <div class="art__photo-container">
+                    <img
+                      src="images/art/2.jpg"
+                      alt="Art photo 2"
+                      class="art__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="art__description">
+              <a
+                href="#"
+                target="_blank"
+                class="art__link"
+              >
+                A New Look at Old Masters
+              </a>
+              <br>
+                December 12, 2020 – Through Spring 2023
+            </div>
+          </div>
+          <div class="gallery__place art">
+            <div class="art__photos">
+              <div class="grid grid--phone">
```

Comment:
images in this section are quite off when testing on resolutions:
![Snapshot 2022-04-13 11 20 31](https://user-images.githubusercontent.com/39308717/163132900-ee335d60-7bd9-4cda-96c8-d709230933a5.png)

Reply:
<img width="408" alt="Screenshot 2022-04-15 at 22 23 55" src="https://user-images.githubusercontent.com/67027882/163649195-cbc35a39-1ca5-45a1-9778-1e5904a976d2.png">
i did the same thing in figma design